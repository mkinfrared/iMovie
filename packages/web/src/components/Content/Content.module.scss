.Content {
  background-color: #000000;
  height: 300vh;
  min-height: calc(100vh - 64px);
  padding: 40px 20px;
  position: relative;

  @media (max-width: 600px) {
    min-height: calc(100vh - 56px);
  }
}
