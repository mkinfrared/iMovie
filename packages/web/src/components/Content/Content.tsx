import Paper from "@material-ui/core/Paper";
import React from "react";

import css from "./Content.module.scss";
import { ContentProps } from "./Content.type";

const Content: React.FC<ContentProps> = ({ children }) => {
  return <Paper className={css.Content}>{children}</Paper>;
};

export default React.memo(Content);
