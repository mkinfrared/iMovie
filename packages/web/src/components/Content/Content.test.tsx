import { render } from "@testing-library/react";
import React, { ReactElement } from "react";

import Content from "./Content";

describe("<Content />", () => {
  let Component: ReactElement;

  beforeEach(() => {
    Component = <Content />;
  });

  it("should be defined", () => {
    const { container } = render(Component);

    expect(container).toBeDefined();
  });

  it("should match a snapshot", () => {
    const { container } = render(Component);

    expect(container).toMatchSnapshot();
  });
});
