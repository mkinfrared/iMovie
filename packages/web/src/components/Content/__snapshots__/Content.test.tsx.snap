// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Content /> should match a snapshot 1`] = `
<div>
  <div
    class="MuiPaper-root Content MuiPaper-elevation1"
  />
</div>
`;
